src/esp/gfx/Renderer.cpp

```diff
@@ -79,7 +82,27 @@ struct Renderer::Impl {
                 esp::gfx::Renderer::Impl::RendererShaderType::
                     DepthTextureVisualizer);
 
+#ifdef CORRADE_TARGET_APPLE
+        Mn::Image2D image = tgt.getDepthTexture().image(
+            0, {Mn::GL::PixelFormat::DepthComponent, Mn::GL::PixelType::Float});
```

Comment:
Like we discussed last week (after the call i think?), this and `setSubImage()` below should be instead something like

```cpp
// after setStorage(), of course
tgt.getFramebuffer().copySubImage({{}, tgt.framebufferSize()}, *visualizedTex_, 0, {});
```

Or, architecture-wise, wrap this in a `RenderTarget::copyFrameDepth()` maybe? [Docs for copySubImage().](https://doc.magnum.graphics/magnum/classMagnum_1_1GL_1_1AbstractFramebuffer.html#a7669d9c7749e58c0462644194433d566)

This will do a much more efficient GPU-GPU copy without going through the CPU. I hope copy from a depth format to a R32F texture works, tho :O If it doesn't work, go through a `BufferImage` instead, which will also make the copy stay on the GPU.